Output/PSIOutput.il

```diff
@@ -0,0 +1,243 @@
+.assembly extern System.Runtime { .publickeytoken = (B0 3F 5F 7F 11 D5 0A 3A) .ver 7:0:0:0 }
+.assembly extern System.Console { .publickeytoken = (B0 3F 5F 7F 11 D5 0A 3A) .ver 7:0:0:0 }
+.assembly extern PSILib { .ver 1:0:0:0 }
+.assembly Comp4 { .ver 0:0:0:0 }
```

Comment:
PSIOutput.dll, PSIOutput.il should not be part of the commit - these files are generated by the compiler and will keep changing every time. Exclude them by adding them to your .gitignore

Reply:
Added the output folder in the .gitignore file